cognite/seismic/protos/types.proto

```diff
@@ -277,3 +277,8 @@ enum IngestionSource {
     FILE_SOURCE = 1;
     TRACE_WRITER = 2;  // TraceWriter
 }
+
+enum Dimensions {
+    TWO_DEE = 0;
+    THREE_DEE = 1;
+}
```

Comment:
Now I feel like a pedant again, but:
I think it's unfortunate that the name was changed to `Dimensions`. A field named `dimensions` seems like it should contain the number of dimensions as an integer, e.g., 2 or 3. But here we define constants 0 and 1 to mean 2D and 3D. Leads to confusion.

If the field and enum were called `dimensionality`, then this would not be as big an issue because a `dimensionality` is not a number, but a semantic category (and hence we use enum constants).

Not sure if it's important enough, but it is not too late to change. No-one is using the interface yet.

Reply:
Would TWO_DEE = 2; THREE_DEE=3; have appeased the physicist, I wonder?

Reply:
;)

Reply:
Maybe :]

Reply:
(I'm the one guilty of s/Dimensionality/Dimensions/ though. I just favored the word with fewer syllables. ;) )

Reply:
Based on the question we had regarding borehole seismic I would suggest to use 0 for 1D, 1 for 2D and 2 for 3D. Until further notice we will have only 1 and 2 but at least we leave place for 1D seismic if it ever comes. 

Does it makes sense to you?

Reply:
I mean, then there's still no correspondence between the constant and the actual number of dimensions, so then I would hesitate to call the field `dimensions`.

Reply:
1 for 1D, 2 for 2D and 3 for 3D?

Reply:
I guess that's the obvious solution :smile: